File: VB/WindowsFormsApplication2/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms

Namespace WindowsFormsApplication2
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnColoredAlert.Click
			alertControl1.Show(Me, "Caption", "Text", "Hot-tracked text", Nothing, Nothing, colorPickEdit1.Color)
		End Sub

        Private Sub btnRegularAlert_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRegularAlert.Click
            alertControl1.Show(Me, "Caption", "Text", "Hot-tracked text")
        End Sub
	End Class
End Namespace

File: CS/WindowsFormsApplication2/Form1.cs
using System;
using System.Windows.Forms;

namespace WindowsFormsApplication2 {
	public partial class Form1 : Form {
		public Form1() {
			InitializeComponent();
		}
        private void simpleButton1_Click(object sender, EventArgs e) {
            alertControl1.Show(this, "Caption", "Text", "Hot-tracked text", null, null, colorPickEdit1.Color);
        }

        private void btnRegularAlert_Click(object sender, EventArgs e) {
            alertControl1.Show(this, "Caption", "Text", "Hot-tracked text", null, null);
        }
	}
}
